site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig

```diff
@@ -95,5 +179,7 @@
         // Only show the settings if the setting is enabled
         onStudentViewChange();
         $('[name="eg_student_view"]').change(onStudentViewChange);
+        $('.eg_config').change(onConfigChange);
+        $('select').change(onConfigPathChange);
```

Comment:
I don't think we want to call this event handler on all `select` element changes